sympy/functions/special/delta_functions.py

```diff
@@ -58,6 +137,54 @@ def fdiff(self, argindex=1):
 
     @classmethod
     def eval(cls, arg, k=0):
+        """
+        Returns a simplified form or a value of DiracDelta depending on the
+        argument passed by the DiracDelta object or the argument ``arg`` passed
+        by the method ``eval()``.
```

Comment:
It is a classmethod so this is self explanatory. 

Something like this, ``Returns a simplified form or a value of DiracDelta depending on the argument passed.`` should suffice.